api/tokens/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import permissions
+
+from website.models import ApiOAuth2PersonalToken
+
+
+class OwnerOnly(permissions.BasePermission):
```

Comment:
This appears to be a direct copy-paste of api/applications/permissions.py, with one word change in the assert statement.

This seems like a prime candidate for some consolidation- this code doesn't need so much duplication.